build.gradle.kts

```diff
@@ -44,7 +44,7 @@ subprojects {
 
         dependencies {
             // BOMs for common projects
-            add("implementation", platform("com.amazonaws:aws-xray-recorder-sdk-bom:2.7.1"))
+            add("implementation", platform("com.amazonaws:aws-xray-recorder-sdk-bom:${version}"))
```

Comment:
Will we always be using the same version of `aws-xray-recorder-sdk-bom` dependency as the agent version? What if the agent version moves forward in the future without the sdk?

Reply:
One of the tenets of the agent is version parity with the SDK. Also, since the SDK is a dependency of the agent and not the other way around, the SDK is always updated first.

Reply:
Okay. But suppose if we need to release a patch v2.8.1 of the agent due to some bug in the agent code. In that case we won't be updating the SDK at all and would still use v2.8.0 of the SDK. Right?

Reply:
That is a good call, darn I merged a little eagerly. I'll revert that quickly.

Reply:
Sorry I approved first and then asked the question.

Reply:
Np :) See #71